src/index.html

```diff
@@ -1,22 +1,422 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap"
+      rel="stylesheet"
+    />
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>BOSE</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/icon-fav.png"
+    />
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="styles/blocks/import.scss"
     />
     <script
       src="scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
```

Comment:
It does not make much sense to have a container inside the header. The header is the container, in this case.

Reply:
This can be said to all the cases on which you used the class "container" in this case. You can just use the outer block as the container.